modules/EvseManager/Charger.hpp

```diff
@@ -228,6 +247,8 @@ class Charger {
 
     const float PWM_5_PERCENT = 0.05;
 
+    const int t_replug = 4000;
```

Comment:
This should have a unit (ms in this case) in its name

Reply:
done